public/js/practise.js

```diff
@@ -60,12 +61,13 @@ function numbersOfLetters(text) {
 
     let letter = text.length;
 
-    console.log(text.length + ' ' + "number of letters in word");
+    return "you have " + text.length + " letter in the word";
 }
+console.log(letters)
 
 // Ex 6 numarul de consoane dintr-un numarul cu functie si parametru//
```

Comment:
The **Ex 6.  __Numarul de consoane dintr-un numarul cu functie si parametru__** has wrong name, please make it correct name and that is **Ex 6. Numarul de Vocale**

Also try to find a solution without using **prompt()** function from **JS**, you can simply write **countVowel('_test_')** and pass in **' '** a string like **_test_** in this example.